### PR TITLE
[WIP] Drawer update

### PR DESCRIPTION
**Pull request by cromefire**
Upgrade for the drawer:
- [ ] Gesture support ([base](https://gist.github.com/developit/a78fceed440b98e62582e4440b86bfab))

### CODE REVIEW

**Comment by artipost** [comment]
Failed pictures (27): [failed-pictures.zip](https://gist.githubusercontent.com/artipost/87029e9f32099634ff6708944f7edf91/raw/a0690914a93f489f436786baeb032f2735c9aae0/failed-pictures.zip)

##
:postbox: Posted by [artipost.io](https://artipost.io), through Travis build [487858288](https://travis-ci.org/prateekbh/preact-material-components/builds/487858288), for commit [a98dc8b](https://github.com/prateekbh/preact-material-components/pull/1277/commits/a98dc8b99978ecf9533ccabdcd9cfaf2d26bf305).

**Comment by cromefire** [comment]
@developit Could you please have a look at the script for the drawer (`packages/drawer/src/index.tsx` and `packages/drawer/src/base.tsx` are the interesting ones)? I didn't get it working.

File: packages/docs/src/components/app/index.js [hunk 1]
```diff
@@ -5,11 +5,14 @@ import Router from 'preact-router';
 // Material Components
 import {FormField} from '@preact-material-components/form-field';
 import {LayoutGrid} from '@preact-material-components/layout-grid';
+import {LayoutGridInner} from '@preact-material-components/layout-grid/lib/inner';
```
[the remark this answers is not in the document]
Comment (cromefire) [reply packages/docs/src/components/app/index.js]: Oh yes, I needed to fix this to be able to start the docs, that's why it's there

[the remark this answers is not in the document]
Comment (cromefire) [reply packages/docs/src/components/app/index.js]: By the way have you any idea how the gesture support could work? I did not get it working...

[the remark this answers is not in the document]
Comment (prateekbh) [reply packages/docs/src/components/app/index.js]: hmmm i'll try to debug take a look in eve

**Comment by prateekbh** [comment]
I took a look at this, but we'll need to work more as this doesn't not drag open the drawer, it just opens it once the drag is complete.

I'll come to this later on

**Comment by cromefire** [comment]
Fixed in a different PR

**Pull request closed without merging**